doc/source/conf.py

```diff
@@ -44,11 +45,36 @@
     'sphinx_gallery.gen_gallery',
 ]
 
+# -- sphinx_gallery configuration
+from sphinx_gallery.gen_gallery import ExplicitOrder
+
+sg_gallery_dir = 'auto_examples'
+sg_examples_dir = '../../examples'
+
+def subsection_title_order(src_dir: str):
+    """
+    src_dir : str
+        string name of source path
+    """
+
+    sub_folders = [p.name for p in Path(src_dir).iterdir() if p.is_dir()]
+    final_order = []
+    for sub_folder in sub_folders:
+        subdir_name = f"{src_dir}/{sub_folder}"
```

Comment:
We should directly use pathlib utilities to manipulate the names rather than doing it manually to ensure we stay portable and robust.

Reply:
Removed